### ISSUE TITLE
Ranked tracking issues.

### ISSUE THREAD

**Issue by gotonickpappas** [Zero-to-Heroes/firestone#319]
I noticed that firestone, does not track ranked games (e.g. wild ranked), and does not record replays. 
This is a recent development. I did not have this problem in the past.
Stats for other modes are tracked properly (e.g. battlegrounds).
The troubleshooting FAQ did not help. Any idea what may be the issue?

**Comment by sebastientromp** [comment]
Could you share a screenshot of your Replays tab?

**Comment by sebastientromp** [comment]
> If this is working as intented then the setting to show the decks in the constructed tab is rather hidden in a different tab.

I agree, I don't think that was intentional. The content of a screen should only be influenced by filters that are on that screen, not things that are in another tab. I will look into it.
